scargo/file_generators/cicd_gen.py

```diff
@@ -46,13 +46,12 @@ def merge_dictionaries(
                 dict1[key] = dict2[key]
         return dict1
 
-    def _get_cicd_content(self, path: str, file_name: str) -> Dict[str, Any]:
-        ci_folder = Path(path)
-        file_path = ci_folder / file_name
+    def _get_cicd_content(self, path: Path, file_name: str) -> Dict[str, Any]:
```

Comment:
Can we give a better name for first argument instead of path? cicd_dir_path or something

Reply:
fixed